server/bundler/babel/babel-loader-cache-identifier/index.js

```diff
@@ -11,7 +11,7 @@ const path = require( 'path' );
  * @return {String}    Module version
  */
 function getModuleVersion( id ) {
-	return require( path.dirname( require.resolve( id ) ) + '/package' ).version;
```

Comment:
You need to put `/[\/\\]lib/` instead of `/[/]lib/` to make this work on Windows.